ProtractorTests/automationTests/specs/logFail.spec.js

```diff
@@ -0,0 +1,13 @@
+var logPage = require('../pages/loginPage'),
+    failPage = require('../pages/failPage');
+
+describe('Login test', function() {
+    it('Enter values for login', function() {
+        logPage.gets('http://www.way2automation.com/angularjs-protractor/registeration/#/login');     
```

Comment:
Why do you send URL as a parameter when your function 'gets()' doesn't have any arguments?